Singleton/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+include_directories(inc)
```

Comment:
Why did you use include_directories and not target_include_directories? Don't modify the code, just answer here :)

Reply:
I usually use it :D